src/backend/app/db/db_models.py

```diff
@@ -168,6 +169,14 @@ class DbOrganisation(Base):
     odk_central_user = cast(str, Column(String))
     odk_central_password = cast(str, Column(String))
 
+    email = cast(str, Column(String))
```

Comment:
Why would the organisation need an email? Is this not something that would be set on the DbUser model?

Reply:
Does the organisation email differ from the user email? It not, we can retrieve from the user, else we need it.

Reply:
The organisation needs an admin, so the org admin user will ideally have something in the `email_address` field.

I don't think we need the email field on organisations.